docs/reference/release-notes/6.8.asciidoc

```diff
@@ -19,6 +19,11 @@ Distributed::
 Machine Learning::
 * Fix restoration of change detectors after seasonality change {ml-pull}1391[#1391]
 
+Security::
+* Fix Elasticsearch field disclosure flaw (ESA-2020-12). (CVE-2020-7019). See
+https://discuss.elastic.co/t/elastic-stack-7-9-0-and-6-8-12-security-update/245456
```

Comment:
@elastic/es-docs should this be something like this: `https://discuss.elastic.co/t/elastic-stack-7-9-0-and-6-8-12-security-update/245456[annoucement in the Elastic forum]` instead?

Reply:
In the past, we've included a link to the PR, and just linked the CVE number to the relevant info. See https://www.elastic.co/guide/en/elasticsearch/reference/7.9/release-notes-7.6.2.html for an example. (Not that it links to elastic.co/community/security, but it doesn't deep link to the specific CVE--linking to the forum post makes more sense to me.)

* Fix Elasticsearch field disclosure flaw (ESA-2020-12) 
https://discuss.elastic.co/t/elastic-stack-7-9-0-and-6-8-12-security-update/245456[CVE-2020-7019]